tests/e2e/test_api.py

```diff
@@ -384,13 +380,15 @@ def test_query_call_with_improper_payload():
         assert "missing" in response.text
 
 
-def test_valid_question() -> None:
+def test_valid_question(qna_data) -> None:
     """Check the REST API /v1/query with POST HTTP method for valid question and no yaml."""
     endpoint = "/v1/query"
+    eval_query, eval_answer = get_eval_question_answer(qna_data, "eval1")
```

Comment:
hmm won't it be possible to use some fixture magic and specify `eval_query` and `eval_answer` as test parameters (via fixture)? https://pytest-with-eric.com/fixtures/pytest-fixture-with-arguments/